src/routes/plans/[id]/+page.svelte

```diff
@@ -240,6 +246,23 @@
   function onChangeRightRowSizes(event: CustomEvent<string>) {
     viewUpdateGrid({ rightRowSizes: event.detail });
   }
+
+  function detectDPRChange() {
```

Comment:
Since this stuff is only used for the Timeline component, could we move this code there? Thinking if we ever wanted to libraritize the Timeline and still have this functionality.

Reply:
Yeah that's a good call.

Reply:
Actually, are you suggesting moving just the detection to the timeline or detection and storage? So removing the device store?

Reply:
Probably more portable to have it all just live in Timeline

Reply:
Yeah I'm good with just having it all in Timeline